tests/integration/modifiers/autofocus-test.js

```diff
@@ -109,4 +109,20 @@ module('Integration | Modifier | autofocus', function(hooks) {
 
     assert.dom('[data-test-textarea="enabled"]').isFocused('The first enabled textarea is focused');
   });
+
+  test('should not focus due to disabled parameter set to true', async function(assert) {
+    await render(hbs`
+      <div {{autofocus 'input:not([disabled]),textarea:not([disabled])' true}}>
+        <span>this is not a focusable element</span>
+        <button data-test-button>this is a button</button>
+        <input data-test-input-1 />
+        <input data-test-input-2 />
+        <input data-test-input-3 />
+      </div>
+    `);
+
+    assert.dom('[data-test-input-1]').isNotFocused('The first non related input are not focused');
```

Comment:
Hey thanks for this PR, I'm glad you took the time to address that 👍

I would just add an assertion on the button (`[data-test-button]`) to check that it is not focused then LGTM.

Reply:
Ok I've added that test! 👍